app/src/main/java/com/iti/android/team1/ebuy/ui/category/CategoryAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.iti.android.team1.ebuy.ui.category
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.iti.android.team1.ebuy.R
+
+class CategoryAdapter : RecyclerView.Adapter<CategoryAdapter.Holder>() {
+
+    class Holder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+          var image : ImageView = itemView.findViewById(R.id.cat_custom_image)
+          var title : TextView = itemView.findViewById(R.id.cat_custom_tv_price)
+          var imgFavorite : ImageView = itemView.findViewById(R.id.cat_custom_img_favo)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): Holder {
+        var view = LayoutInflater.from(parent.context).inflate(R.layout.category_custom_rv_item,null,false)
```

Comment:
we use view binding so you can get all view directly

Reply:
ok